xml/System.Net/ServicePoint.xml

```diff
@@ -30,11 +30,11 @@
 ## Remarks  
  The <xref:System.Net.ServicePoint> class handles connections to an Internet resource based on the host information passed in the resource's Uniform Resource Identifier (URI). The initial connection to the resource determines the information that the <xref:System.Net.ServicePoint> object maintains, which is then shared by all subsequent requests to that resource.  
   
- <xref:System.Net.ServicePoint> objects are managed by the <xref:System.Net.ServicePointManager> class and are created, if necessary, by the <xref:System.Net.ServicePointManager.FindServicePoint%2A?displayProperty=nameWithType> method. <xref:System.Net.ServicePoint> objects are never created directly but are always created and managed by the <xref:System.Net.ServicePointManager> class. The maximum number of <xref:System.Net.ServicePoint> objects that can be created is set by the <xref:System.Net.ServicePointManager.MaxServicePoints%2A?displayProperty=nameWithType> property.  
+ <xref:System.Net.ServicePoint> objects are managed by the <xref:System.Net.ServicePointManager> class and are created, if necessary, by the <xref:System.Net.ServicePointManager.FindServicePoint?displayProperty=nameWithType> method. <xref:System.Net.ServicePoint> objects are never created directly but are always created and managed by the <xref:System.Net.ServicePointManager> class. The maximum number of <xref:System.Net.ServicePoint> objects that can be created is set by the <xref:System.Net.ServicePointManager.MaxServicePoints?displayProperty=nameWithType> property.  
```

Comment:
The link to FindServicePoint is broken; the `%2A` is necessary here to link to the overload list, rather than a specific overload.

Reply:
Ah, I didn't think the %2A is useful at all, sorry for that. Is there a place I can review which of them are now broken?

Reply:
Here is the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C7%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C201804071856598956-30%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ERp1OExM%2FAbLwCGOQZn8k1sOKPbGikIhEYqaCg7Gkj8%3D&st=2018-04-07T20%3A17%3A56Z&se=2018-05-08T20%3A22%3A56Z&sp=r). If you expand "All checks have passed" below, the Details links for the OpenPublishing.Build item leads to the report. And this is the staging link to the [ServicePoint documentation](https://review.docs.microsoft.com/en-us/dotnet/api/system.net.servicepoint?view=netframework-4.7.2&branch=pr-en-us-30).